src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -0,0 +1,1152 @@
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="infra-events"
+    title="Infrastructure events"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+    Most infrastructure events depend on inventory or custom events pushed by the New Relic infrastructure agent.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="third-party-svc"
+    title="Third-party services"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+    [On-host integrations](/docs/infrastructure/infrastructure-integrations/get-started/introduction-infrastructure-integrations#on-host) (NGINX, MySQL, Redis, Apache, RabbitMQ, and many more) and the [Kubernetes integrations](/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/) are required as a source for curated dashboards and entities in New Relic.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="k8s-integ"
+    title="Application/service integration with K8s telemetry"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+    New Relic agent data for services shows up in the Kubernetes cluster explorer, but OpenTelemetry services data does not yet. We recommend that for any OpenTelemetry service using Kubernetes you should decorate your telemetry with the relevant resource attributes from the OpenTelemetry semantic conventions so you can query for this data.
+  </Collapser>
+</CollapserGroup>
+
+## Serverless [#serverless]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry:  🟡 Limited support
+
+OpenTelemetry serverless function telemetry can be reported, stored, and visualized with custom dashboards. A dedicated view of this telemetry is not yet available.
+
+## Alert [#alerting]
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="alerts-collap"
+    title="Alerts"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅ 
+
+    You can configure alerts on OpenTelemetry data using conditions written in NRQL. Click any chart’s “...” menu to get started.
+
+    Conditions, violations, incidents, and issues all work for any entity.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="recommended-conditions"
+    title="Recommended alert conditions"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="alert-condit-list"
+    title="Alert conditions list UI"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅ 
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="alert-viol-list"
+    title="Alert violations list UI"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+  </Collapser>
+</CollapserGroup>
+
+## New Relic platform
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="logs-context"
+    title="Logs in context"
```

Comment:
Add a feature "Logs in context: Automatic logs ingest", with a "yes" for NR agent and a "no" for OTel